backend/src/controllers/question.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const router = express.Router({mergeParams: true})
```

Comment:
It's because we want to be able to access cid from within the question route. Since cid comes through the course route it becomes inaccessible. Supposedly this merges the routes so that subroutes get access to the full path or something like that at least.